packages/cli/commands/accounts/list.js

```diff
@@ -22,29 +29,32 @@ exports.describe = i18n(`${i18nKey}.describe`);
 
 const sortAndMapPortals = portals => {
   const mappedPortalData = {};
+  // Standard and app developer portals
   portals
-    .sort((a, b) => {
-      if (a.sandboxAccountType === null && b.sandboxAccountType !== null) {
-        return -1;
-      }
-      if (a.sandboxAccountType !== null && b.sandboxAccountType === null) {
-        return 1;
-      }
-      return 0;
-    })
+    .filter(p =>
+      p.accountType
+        ? p.accountType === HUBSPOT_ACCOUNT_TYPES.STANDARD ||
+          p.accountType === HUBSPOT_ACCOUNT_TYPES.APP_DEVELOPER
+        : p.sandboxAccountType === null
+    )
     .forEach(portal => {
-      if (
-        portal.sandboxAccountType !== undefined &&
-        portal.sandboxAccountType === null
-      ) {
-        mappedPortalData[portal.portalId] = [portal];
-      } else if (portal.sandboxAccountType && portal.parentAccountId) {
-        mappedPortalData[portal.parentAccountId] = [
-          ...(mappedPortalData[portal.parentAccountId] || []),
-          portal,
+      mappedPortalData[portal.portalId] = [portal];
+    });
+  // Non-standard portals (sandbox, developer test account)
+  portals
+    .filter(p =>
+      p.accountType
+        ? isSandbox(p) || isDeveloperTestAccount(p)
+        : p.sandboxAccountType !== null
+    )
+    .forEach(p => {
+      if (p.parentAccountId) {
+        mappedPortalData[p.parentAccountId] = [
+          ...(mappedPortalData[p.parentAccountId] || []),
+          p,
```

Comment:
All backwards compatible with older configs without `accountType`, however the validation PR (https://github.com/HubSpot/hubspot-local-dev-lib/pull/105) just got merged so these should be populated as soon as any command with a validation step is run